src/reconcile/rec_write.c

```diff
@@ -3229,6 +3229,18 @@ __rec_col_int(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_PAGE *page)
 	WT_RET(__rec_split_init(
 	    session, r, page, page->pg_intl_recno, btree->maxintlpage));
 
+	/*
+	 * We need to mark this page as splitting, as in memory splits are
+	 * now permitted
```

Comment:
Now compared to earlier versions of the code? It's best to describe the current behavior without reference to old versions of the code. I don't think this block of code needs a comment - it's clear that we are switching to splitting from the code.